contracts/cw-core/src/contract.rs

```diff
@@ -181,19 +189,67 @@ pub fn execute_pause(
         .add_attribute("until", until.to_string()))
 }
 
+pub fn execute_admin_msgs(
```

Comment:
What do you think about letting the admin execute a message on a paused DAO? Think that could make sense in the theme of "admin" can control the subDAO.

Reply:
this also could prevent some bricked sub DAO situations!

Reply:
this also could prevent some bricked sub DAO situations!